extensions/wikia/Search/classes/IndexService/DefaultContent.php

```diff
@@ -229,30 +258,35 @@ protected function prepValuesFromHtml( $html ) {
 			$plaintext = $this->getPlaintextFromDom( $dom );
 			$paragraphs = $this->getParagraphsFromDom( $dom );
 		} else {
-			$plaintext = html_entity_decode( strip_tags( $html ), ENT_COMPAT, 'UTF-8' );			
+			$plaintext = html_entity_decode( strip_tags( $html ), ENT_COMPAT, 'UTF-8' );
 		}
-		$plaintext = trim(preg_replace(static::SPACE_SEQUENCE_REGEXP, ' ', $plaintext));
-		$paragraphString = trim(preg_replace( static::SPACE_SEQUENCE_REGEXP, ' ', implode( ' ', $paragraphs ) )); // can be empty
-		$words = preg_split( '/ /', $paragraphString?: $plaintext);
+		$plaintext = trim( preg_replace( static::SPACE_SEQUENCE_REGEXP, ' ', $plaintext ) );
+		$paragraphString =
+			trim( preg_replace( static::SPACE_SEQUENCE_REGEXP, ' ', implode( ' ', $paragraphs ) ) ); // can be empty
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)